src/components/Header.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import Header from './Header'
+
+test('renders test text', () => {
```

Comment:
Do we want to use `test` or `it` convention? :thinking:
```suggestion
it('should render test text', () => {
```

Reply:
The difference is insignificant, I'd prefer "test" as it makes it easier to understand. Testing something > test > less words to remember > less pain.

Reply:
As far as I understand the difference between using 'it' or 'test', I find the difference between them minimal/inconsequential. The Jest documentation even states they 'it' is an 'alias' version of 'test'. 

`describe('yourModule', () => {`
`  test('if it does this thing', () => {});`
`  test('if it does the other thing', () => {});`
`});`

`describe('yourModule', () => {`
`  it('should do this thing', () => {});`
`  it('should do the other thing', () => {});`
`});` 

I would prefer using 'test' because it seems more natural and foolproof to me. What do I want to do? I want to make a test. What do I want to do? It should do this. Well... Maybe it's more readable for seniors who wouldn't even think the code could be anything other than a test just based on a look of it, but for me, seeing it literally reads as "this is a test" makes it much more readable and easy to understand.